app/views/planning_application/review_policy_classes/edit.html.erb

```diff
@@ -39,60 +39,21 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
+    <%= render(
+      partial: "shared/policy_classes/table",
```

Comment:
I've moved the policy classes `table` element out of the `form` element, because I need to nest the comment `form` elements inside it, and you can't nest a form within a form!

Reply:
Ah... I knew a form within form was a bad thing but I didn't have any answer to the problem.